src/bundle/Resources/public/js/scripts/admin.contenttype.edit.js

```diff
@@ -190,17 +190,28 @@
             }
         });
     };
-    const addField = ({ targetContainer, draggedItemPosition }) => {
+    const addField = ({ targetContainer, draggedItemPosition, fieldRendered }) => {
         if (!sourceContainer.classList.contains('ibexa-available-field-types__list')) {
-            insertFieldDefinitionNode(currentDraggedItem, { targetContainer, draggedItemPosition });
+            const insertedField = insertFieldDefinitionNode(currentDraggedItem, { targetContainer, draggedItemPosition });
```

Comment:
In all places where this kind of variable is added (returned value from `insertFieldDefinitionNode` or `createFieldDefinitionNode`) is mostly cosmetic change - only in one place it really matters, but I wanted to be coherent in all places.

This new variable is used in all places that manipulates DOM of field (in most cases element passed as first element is same element as returned value (unless you pass string html instead of DOM element - https://github.com/ibexa/admin-ui/blob/f4bd86034dcaeb3dece0daaab465e79a206aaa03/src/bundle/Resources/public/js/scripts/admin.contenttype.edit.js#L79)